blocks/page_list/templates/worldskills_news_list.php

```diff
@@ -3,8 +3,18 @@
 $th = Loader::helper('text');
 
 $thumbnailType = \Concrete\Core\File\Image\Thumbnail\Type\Type::getByHandle('landscape');
+
+$app = \Concrete\Core\Support\Facade\Application::getFacadeApplication();
+$canonical = $app->make(Concrete\Core\Url\SeoCanonical::class);
+$tag = $canonical->getPathArguments()[1];
 ?>
 
+<h1 class="ws-h-icon h2"><?php
+    if (isset($pageListTitle) && $pageListTitle) echo h($pageListTitle); else echo 'News';
```

Comment:
Can you change this so it only shows an `<h1>` if `$pageListTitle` is set? That way we can leave away the title if we want by just not setting a page title. More flexible. So just move the `if () {` to line 11 before `<h1>`.

Reply:
okay, to confirm, so no need to have a default 'News' text as the title if we don't set the page title?

Reply:
Correct, no need for a default title.